src/lib/discord/commando/commands/poracle-clean.js

```diff
@@ -8,15 +8,15 @@ exports.run = async (client, msg) => {
 		const limit = 100
 		const messages = await msg.channel.messages.fetch({ limit })
 
-		const startMessage = await msg.reply(`Will start cleaning up to ${limit} messages back - do not re-run until finished`)
+		const startMessage = await msg.reply(`${client.translator.translate('Will start cleaning up to')} ${limit} ${client.translator.translate('messages back - do not re-run until finished')}`)
```

Comment:
Thank you for this!
Did you verify that this works (this is a pure discord command rather than having all the poracle infrastructure available so couldn't tell you off the top of my head)
If translateFormat is available here then 'Will start cleaning up to {0} messages etc' might be better than having two strings